README Projeto Álcool.bet

Brenopy opened the pull request:
Arquivo README.md com as informações do projeto Álcool.bet da disciplina Projeto Integrador.

Brenopy committed:
README Projeto Álcool.bet

gabrielpatzer commented:
O joão pedro não faz parte do grupo?
Inclua também o link do projeto de vocês no nome e, se possível, o link do github de cada integrante nos nomes.

Brenopy committed:
Update README.md

Adicionado o link do projeto e dos participantes (acrescentando o João Pedro Gambim).

gabrielpatzer commented:
é preciso corrigir a sintaxe dos links. Ao adicionar o ! antes do texto o MD entende como uma imagem ao invés de link.
[sintaxe básica de MD](https://www.markdownguide.org/basic-syntax/)

Brenopy committed:
Update README.md

Links arrumados 🙏🙏🙏
